com.unity.shadergraph/Documentation~/Keywords.md

```diff
@@ -1,50 +1,62 @@
 | **Display Name**   | String   | The display name of the Keyword. Unity shows this name in the title bar of nodes that reference the corresponding Keyword, and also in the Material Inspector if you expose that Keyword. |
-| **Exposed**        | Boolean  | If you set this to **true**, Unity displays the corresponding Keyword in the Material Inspector. If you set it to **false**, the Keyword does not appear in the Material Inspector. |
+| **Exposed**        | Boolean  | When you set this parameter to **true**, Unity displays this Keyword in the Material Inspector. If you set it to **false**, the Keyword does not appear in the Material Inspector. |
 | **Reference Name** | String   | The internal name for the Keyword in the shader.<br/><br/>If you overwrite the Reference Name parameter, take note of the following:<br/>&#8226; Keyword Reference Names are always in full capitals, so Unity converts all lowercase letters to uppercase.<br/>&#8226; If the Reference Name contains any characters that HLSL does not support, Unity replaces those characters with underscores.<br/>&#8226; Right-click on a Reference Name, and select **Reset Reference** to revert to the default Reference Name. |
-| **Definition**     | Enum     | Sets how the Keyword is defined in the shader.<br/><br/>There are three available options.<br/>&#8226; **Shader Feature**: Unity strips unused shader variants at build time.<br/>&#8226; **Multi Compile**: Unity never strips any shader variants.<br/>&#8226; **Predefined**: Indicates that the active Render Pipeline has already defined this Keyword, so Shader Graph does not define it in the code it generates. |
+| **Definition**     | Enum     | Sets how the Keyword is defined in the shader. Determines when to compile keyword variants.
+<br/><br/>There are three available options.<br/>&#8226; **Shader Feature**: Unity only compiles keyword variants when a Material selects the relevant option. For this option to be available in the Player, a Material selecting it must exist at build-time.<br/>&#8226; **Multi Compile**:Pre-compiles all the variant possibilities. This is slower and uses more memory, but allows the option to be dynamically switched in the Player.
+<br/>&#8226; **Predefined**: The render pipeline defines this keyword and controls the settings for it. |
 | **Scope**          | Enum     | Sets the scope at which to define the Keyword.<br/><br/>&#8226; **Global Keywords**: Defines Keyword for the entire project, and it counts towards the global keyword limit.<br/>&#8226; **Local Keywords**: Defines Keyword for only one shader, which has its own local keyword limit.<br/><br/>When you use Predefined Keywords, Unity disables this field. |
+| **Stages** |  | <br/>&#8226; **All** - Applies this keyword to all shader stages.
+<br/>&#8226;**Vertex** - Applies this keyword to the vertex stage.
+<br/>&#8226;**Fragment** - Applies this keyword to the fragment stag
```

Comment:
End of the sentence here is clipped short (should read "... the fragment stage."

Reply:
fixed.